### DISCUSSION TITLE
Can I show complete number of stats instead of `xx`k?

### DISCUSSION THREAD

**Discussion by lyc8503** [anuraghazra/github-readme-stats#3445] [Q&A] [answered]
I am using the stats card on [my profile](https://github.com/lyc8503) and I have just reached 1k stars.

This stats card works well expect that it's showing my star counts as `1k` instead of `1000`.

I am wondering if I can choose to show the complete number instead of `k`.

**Comment by qwerty541** [comment, marked as answer]
Hey, @lyc8503! You can use `&number_format=long` querystring parameter for this purpose. You can also remove `&count_private=true` parameter from your URL since it is outdated and does not affect on the card. Currently to show your private contributions you should [deploy own Vercel instance](https://github.com/anuraghazra/github-readme-stats#on-vercel) using own GitHub API token.

```
https://github-readme-stats.vercel.app/api?username=lyc8503&layout=compact&show_icons=true&number_format=long
```

<img src="https://github-readme-stats.vercel.app/api?username=lyc8503&layout=compact&show_icons=true&number_format=long" />
